src/kristin_tukun_ex/ex01/letter_counts.py

```diff
@@ -0,0 +1,19 @@
+def letter_freq(txt):
+    """
+    This is a function with text as input, it will then count occurrence of letters and signs in the text. It does not
+    differentiate on upper an lowercase letters.
+    :param txt: A text string
+    :return: A list of number of occurrences.
+    """
+    from collections import Counter
+    letters = txt.lower()
+    letters = sorted(letters)
```

Comment:
Nice think! However, as stated:

> As an extra challenge, modify the code printing the table so that letters are printed in alphabetical order

sorted function should be in the main code, not here.
for letter, count in sorted(frequencies.items()):